server/src/connectionHandlers/onCompletion.ts

```diff
@@ -268,6 +271,24 @@ function getFilePath (elementInfo: ElementInfo, fileType: string): string | unde
   return undefined
 }
 
+function convertSymbolsToCompletionItems (uri: string): CompletionItem[] {
+  logger.debug(`[onCompletion] convertSymbolsToCompletionItems: ${uri}`)
+  const completionItems: CompletionItem[] = []
+  analyzer.getSymbolsForUri(uri).forEach((symbol) => {
+    const completionItem: CompletionItem = {
+      label: symbol.symbolName,
+      labelDetails: {
+        description: 'Source: Symbol Scanner'
+      },
+      documentation: '',
```

Comment:
As illustrated in your screenshots, we get double the symbols for completion. It would be better to have just one completion item and fill it with the data from both doc and symbol analysis.

Reply:
Done. See [190f52295cb5106b0b1ab1f2f90deb3ce8f40a6e](https://github.com/savoirfairelinux/vscode-bitbake/pull/99/commits/190f52295cb5106b0b1ab1f2f90deb3ce8f40a6e)